### PR TITLE
Fix: add pagination to user api keys

### PR DESCRIPTION
**Pull request by oyershov**

### CODE REVIEW

File: src/containers/ProfileApiKeys/index.tsx [hunk 1]
```diff
@@ -72,12 +85,21 @@ class ProfileApiKeysComponent extends React.Component<Props, ProfileApiKeysState
         }
     };
 
-    public componentDidMount(): void {
-        this.props.getApiKeys();
+    public componentDidMount() {
+        this.props.apiKeysFetch({ pageIndex: 0, limit: 4 });
```
Comment (akhlopiachyi) [inline src/containers/ProfileApiKeys/index.tsx:89]: take limit from configs

File: src/containers/ProfileApiKeys/index.tsx [hunk 2]
```diff
@@ -115,10 +137,20 @@ class ProfileApiKeysComponent extends React.Component<Props, ProfileApiKeysState
                 )}
 
                 {user.otp && dataLoaded && apiKeys.length > 0 && (
-                    <Table
-                        header={this.getTableHeaders()}
-                        data={apiKeys && apiKeys.length ? this.getTableData(apiKeys) : [[]]}
-                    />
+                    <>
```
Comment (akhlopiachyi) [inline src/containers/ProfileApiKeys/index.tsx:140]: Better use <React.Fragment>
Some errors can appear

File: src/containers/ProfileApiKeys/index.tsx [hunk 3]
```diff
@@ -159,7 +191,7 @@ class ProfileApiKeysComponent extends React.Component<Props, ProfileApiKeysState
                         <Form>
                             <Form.Check
                                 type="switch"
-                                id="apiKeyCheck"
+                                id={`apiKeyCheck${item.kid}`}
```
Comment (akhlopiachyi) [inline src/containers/ProfileApiKeys/index.tsx:194]: Make id like this: apiKeyCheck-${item.kid}

File: src/containers/ProfileVerification/index.tsx [hunk 4]
```diff
@@ -154,17 +159,18 @@ class ProfileVerificationComponent extends React.Component<Props, State> {
...
                         >
-                            <Link to="/confirm"><FormattedMessage id="page.body.profile.verification.reverify" /></Link>
-                            <CrossIcon />
-                        </div>
-                        <span className={tooltipClass}>
-                            <FormattedMessage id={`page.body.profile.verification.${labelToCheck}.rejected.tooltip`} />
-                        </span>
+                            <div
+                                className="pg-profile-page-verification__step__label pg-profile-page-verification__step__label--rejected"
+                            >
```
Comment (akhlopiachyi) [inline src/containers/ProfileVerification/index.tsx:169]: write it in one line

File: src/containers/ProfileApiKeys/index.tsx [hunk 5]
```diff
@@ -115,10 +137,20 @@ class ProfileApiKeysComponent extends React.Component<Props, ProfileApiKeysState
...
 
                 {user.otp && dataLoaded && apiKeys.length > 0 && (
-                    <Table
-                        header={this.getTableHeaders()}
-                        data={apiKeys && apiKeys.length ? this.getTableData(apiKeys) : [[]]}
-                    />
+                    <React.Fragment>
+                        <Table
+                            header={this.getTableHeaders()}
+                            data={apiKeys && apiKeys.length ? this.getTableData(apiKeys) : [[]]}
```
Comment (josadcha) [inline src/containers/ProfileApiKeys/index.tsx:143]: apiKeys is an array so no need to check if it is defined

**Review by josadcha — CHANGES_REQUESTED** [review]

**Review by josadcha — APPROVED** [review]

**Review by akhlopiachyi — APPROVED** [review]

**Pull request merged**